tests/test_util.py

```diff
@@ -175,3 +176,21 @@ def test_compute_groups_from_parameter_file():
     assert_equal(actual_matrix, np.array(
         [[1, 0], [0, 1], [0, 1]], dtype=np.int))
     assert_equal(actual_unique_names, ['Group 1', 'Group 2'])
+
+def test_nonuniform_scale_samples_truncnorm():
+    """
+    Test the rescaling of samples for truncated normal distribution
+    """
+    problem = {
+        'num_vars': 1,
+        'bounds': [[0, 3.14, 2, 1]],
+        'names': ['x1']
```

Comment:
Test seems to be missing the `dists` entry

e.g. `'dists': 'truncnorm'`

Just in case, you can run this specific test locally by running the below from the project directory:

`pytest tests/test_util.py::test_nonuniform_scale_samples_truncnorm`